recipes/dbus-cxx/2.x.x/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.files import get
+from conan.tools.scm import Version
+
+class DbusCXX(ConanFile):
+    name = "dbus-cxx"
+    license = "LGPL-3.0-only", "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://dbus-cxx.github.io"
+    description = "DBus-cxx provides an object-oriented interface to DBus"
+    topics = "bus", "interprocess", "message"
+    package_type = "static-library"
```

Comment:
is this correct? this seems to be unconditionally shared
https://github.com/dbus-cxx/dbus-cxx/blob/master/CMakeLists.txt#L250